collections/src/main/scala/strawman/collection/immutable/LongMap.scala

```diff
@@ -439,4 +447,7 @@ sealed abstract class LongMap[+T] extends Map[Long, T]
     case LongMap.Nil => throw new IllegalStateException("Empty set")
   }
 
+  def map[V2](f: ((Long, T)) => (Long, V2)): LongMap[V2] = LongMap.from(View.Map(coll, f))
+
+  def flatMap[V2](f: ((Long, T)) => IterableOnce[(Long, V2)]): LongMap[V2] = LongMap.from(View.FlatMap(coll, f))
```

Comment:
IntMap also overrides `concat`, it would probably make sense to do the same here.